index.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+	env: {
+		browser: true,
+		es6: true,
+	},
+	plugins: ['jsdoc', 'prettier'],
+	extends: ['airbnb-base', 'prettier', 'plugin:prettier/recommended'],
```

Comment:
@nicholasio I have concerns about promoting and integrating the work of another company without fully vetting the culture. You never know what associations could crop up in the future. I'd be more open to forking or using it as a starting point rather than pulling it in and turning off the stuff we don't want TBH (@tlovett1 )

Reply:
I don't see a problem with integration work from another company since that's the spirit of open-source. 

I'm considering the airbnb rules to be a starting point. I'm disabling or changing rules according to our needs/culture.

But I'm open to forking (copying all rules) to this config.

Reply:
I wouldn't think this would be a problem. The Airbnb config seems to be the de-facto open source go-to, and doesn't appear to have anything to do with Airbnb, other than it's developers there that maintain it. Seems analogous to React coming from Facebook. Except that Airbnb actually has their name in the title of the package and forking it wouldn't be as big a maintenance burden.

Reply:
@timwright12 appreciate your concern. Right now we use software from other people/companies in almost everything we do. I don't know if it makes sense to open the pandora's box of needing to review culture of those people/companies. I'm fine with using Airbnb here.

That said, I can see a world where we do take a strong stance on a particular company/person and rethink the approach in that instance.